.gitignore

```diff
@@ -13,14 +13,16 @@ cypress.zip
 Cached Theme.pak
 Cached Theme Material Design.pak
 
+# from config
+packages/config/src/*.js
```

Comment:
I might be confused, but don't you want to ignore `lib`, not `src`?

Reply:
I think I fixed it this round. Theres are some expected scripts I missed the first round for local development: [`953dacd` (#18589)](https://github.com/cypress-io/cypress/pull/18589/commits/953dacd0e216d05082fa629e6da4fd2f520c6b5e)